Archive.Zip.UnitTests/ZipFileBuilderTest.cs

```diff
@@ -675,46 +699,46 @@ public void Build_UsesProperFileShareToOpenTheFile (FileShare specifiedAdditionF
     private Func<IDirectoryInfo, IFileInfo> CreateFile (string name, int size)
     {
       return parent => new InMemoryFileInfo (
-          Path.Combine (parent.FullName, name),
-          new NotClosableMemeoryStream (new byte[size]),
-          parent,
-          DateTime.Now,
-          DateTime.Now,
-          DateTime.Now);
+                 Path.Combine (parent.FullName, name),
+                 new NotClosableMemeoryStream (new byte[size]),
+                 parent,
+                 DateTime.Now,
+                 DateTime.Now,
+                 DateTime.Now);
     }
 
     private Func<IDirectoryInfo, InMemoryDirectoryInfo> CreateDirectory (
         string name,
         params Func<IDirectoryInfo, IFileSystemEntry>[] subEntries)
     {
       return parent =>
-      {
-        var directory = new InMemoryDirectoryInfo (
-            parent != null ? Path.Combine (parent.FullName, name) : name,
-            parent,
-            DateTime.Now,
-            DateTime.Now,
-            DateTime.Now);
-
-        foreach (var subEntry in subEntries)
-        {
-          var value = subEntry (directory);
-
-          var dir = value as IDirectoryInfo;
-          if (dir != null)
-          {
-            directory.Directories.Add (dir);
-          }
-          else
-          {
-            var file = value as IFileInfo;
-            if (file != null)
-              directory.Files.Add (file);
-          }
-        }
-
-        return directory;
-      };
+             {
+               var directory = new InMemoryDirectoryInfo (
+                   parent != null ? Path.Combine (parent.FullName, name) : name,
+                   parent,
+                   DateTime.Now,
+                   DateTime.Now,
+                   DateTime.Now);
+
+               foreach (var subEntry in subEntries)
+               {
+                 var value = subEntry (directory);
+
+                 var dir = value as IDirectoryInfo;
+                 if (dir != null)
+                 {
+                   directory.Directories.Add (dir);
+                 }
+                 else
+                 {
+                   var file = value as IFileInfo;
+                   if (file != null)
+                     directory.Files.Add (file);
+                 }
+               }
+
+               return directory;
+             };
```

Comment:
Unnecessary whitespace change

Reply:
How many of those whitespace-changes are due to the migration tool?